types/js-crc/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for js-crc 0.2
+// Project: https://github.com/emn178/js-crc
+// Definitions by: Anders Kjeldsen <https://github.com/and3k5>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function crc32(message: string | ArrayBuffer | Uint8Array | ReadonlyArray<number>): string;
+
+export function crc16(message: string | ArrayBuffer | Uint8Array | ReadonlyArray<number>): string;
+
+export as namespace jsCrc;
```

Comment:
Don't think this is quite right for the global case - the library readme says it patches `crc32` and `crc16` directly onto the global object. I'd just leave this off unless the globals actually matter to you and, if they do, you should remove this and add a `globals.d.ts` that isn't loaded by default that specifies the global types.

Reply:
You're right. I dont know why I left it there.

For now I'll remove that particular line, test again and push.